utils/rowgroup/rowgroup.h

```diff
@@ -61,11 +61,15 @@
 #include "collation.h"
 #include "common/hashfamily.h"
 
+#include "stdlib.h"
+#include "execinfo.h"
+
 // Workaround for my_global.h #define of isnan(X) causing a std::std namespace
 
 namespace rowgroup
 {
 const int16_t rgCommonSize = 8192;
+const uint32_t rgNullOffset = std::numeric_limits<uint32_t>::max();
```

Comment:
This const is not used in the patch plz remove it.

Reply:
Done.